src/commands/scan.yml

```diff
@@ -67,13 +72,21 @@ parameters:
     description: Elapsed time the command can run without output. The default is 10 minutes.
     type: string
     default: "10m"
+  test-param-fail-on-no-cache-used:
```

Comment:
suggestion: let's add experimental or something similar to make it even more clear to not use it

Reply:
it's not experimental - but I'll rename it to `test-only_...`